cnf-certification-test/identifiers/identifiers.go

```diff
@@ -908,6 +909,22 @@ tag. (2) It does not have any of the following prefixes: default, openshift-, is
 		},
 		TagCommon)
 
+	TestOperatorHasSemanticVersioningIdentifier = AddCatalogEntry(
+		"installed-operator-semantic-versioning",
```

Comment:
I'd just call it "semantic-versioning". This way when prefixed with the suite name "operator" the test case name would simply be "operator-semantic-versioning" instead of "operator-installed-operator-semantic-versioning".

Reply:
The Operators need to have a label or belong to a namespace defined in the `tnf_config.yml` to be tested, so I think there should be no confusion.

Reply:
No, we are considering the operators here irrespective of labels or namespace.

Reply:
But before you said this only applies to non-platform operators. If you don't take into account the labels or the namespace you are testing all operators in the cluster.